languages/elixir/exercises/concept/processes/.docs/after.md

```diff
@@ -0,0 +1,133 @@
+In Elixir, all code runs inside processes. Elixir processes:
+
+- Should not be confused with system processes.
+- Are lightweight.
+  - It is normal to have an Elixir app that runs hundreds of processes.
+- Have specific use cases.
+  - Processes can:
+    - [Keep global state][getting-started-processes-state].
+    - [Contain failure][getting-started-processes-links].
+    - Allow for concurrent and asynchronous code.
+  - It is also normal to have an Elixir app that doesn't explicitly create new processes, especially if it's a library.
+
+## Creating processes
+
+By default, a function will execute in the same process from which it was called. When you need to explicitly run a certain function in a new process, use `spawn`:
+
+- [`spawn/1`][kernel-spawn-1] accepts a function that it will execute directly.
+
+  ```elixir
+  spawn(fn -> 2 + 2 end)
+  # => #PID<0.125.0>
+  ```
+
+- [`spawn/3`][kernel-spawn-3] accepts a function that it will execute by the module name, the function name (as atom), and a list of arguments to pass to that function.
+
+  ```elixir
+  spawn(String, :split, ["hello there", " "])
+  # => #PID<0.113.0>
+  ```
+
+  - This data triplet is often called _MFA_ - _Module, Function, Arguments_.
+
+- A process exists as soon as its function finished executing.
+
+  - You can check if a process is still _alive_ (executing) with [Process.alive?/1][process-alive]:
+
+    ```elixir
+    pid = spawn(fn -> 2 + 2 end)
+    Process.alive?(pid)
+    # => false
+    ```
+
+## Messages
+
+Processes do not directly share information with one another. The only way to get data in and out of a running process is by _sending messages_. This type of concurrency is called the [Actor model][wiki-actor-model].
+
+- You can send messages to a process using [send/2][kernel-send].
+
+  ```elixir
+  send(pid, :hello)
+  ```
+
+  - The message ends up in the recipient's _mailbox_.
```

Comment:
```suggestion
  - The message ends up in the recipient's _mailbox_ in the order that they are sent.
```

Reply:
In addition to my suggestion, should we mention that if multiple processes are sending messages to a process, there is no guarantee about which order the messages are interleaved?